src/ray/rpc/grpc_server.h

```diff
@@ -32,8 +33,8 @@ class GrpcServer {
   ///  will be chosen.
   /// \param[in] main_service The main event loop, to which service handler functions
   /// will be posted.
-  GrpcServer(const std::string &name, const uint32_t port)
-      : name_(name), port_(port), is_closed_(true) {}
+  GrpcServer(std::string name, const uint32_t port)
+      : name_(std::move(name)), port_(port), is_closed_(true) {}
```

Comment:
The previous code seemed clearer to me, why did you change it?

Reply:
Clang-tidy suggests this change. Passing by value + move has some advantages over passing by const reference + copy. 
The biggest advantage is that you can avoid copying data.  E.g.,
```
std::string name = "....";
GrpcServer(std::move(name));  // no copy at all.
```

See https://stackoverflow.com/questions/51705967/advantages-of-pass-by-value-and-stdmove-over-pass-by-reference/51706522 for more details.

Reply:
Ok interesting, thanks for the explanation :)